.env.sample

```diff
@@ -0,0 +1,2 @@
+export PORT=6060
```

Comment:
@manusajith I think this file belongs to the `starter` branch

Reply:
Needed a rebase here. Should be good now.